### ISSUE TITLE
Feature Request: Support for MacOS and Linux 

### ISSUE THREAD

**Issue by rafaga** [Slazanger/SMT#93]
As part of the migration to .net core 6, should be great to have a version compatible with Linux / MacOS with out using Wine or something like that

this should deprecate the bug [#67](https://github.com/Slazanger/SMT/issues/67)

**Comment by martinskalicky** [comment]
++

**Comment by lorensghassan** [comment]
I am interested in contributing towards the completion of this item; what work remains to be done?

**Comment by lorensghassan** [comment]
@rafaga I am looking over the dll.config file for the compiled binary for the zip file, it looks like it is target .NET 4.8:

'<supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8" />'

Although, perhaps there is a newer build than two weeks that is .NET Core.

Can you associate the PR that adds MacOS support to this issue as well? I went through the PRs and did not see the one that adds this functionality.

**Comment by rafaga** [comment]
sorry, I think you should talk with @Slazanger (the author) about the current state of the development. Currently I'm winning at eve, so I can't help much right now

**Comment by lorensghassan** [comment]
@rafaga You closed the issue. If its not resolved, why would you close the issue? I will reopen.

**Comment by lorensghassan** [comment]
@Slazanger If this issue is not resolved, please re-open. I am interested in contributing to build out a .NET Core version (not just compatible, but .NET Core binary) / MacOS native compatible version if this issue is not resolved.

**Comment by inspxctelement** [comment]
bump. wouldn't mind helping out getting the OSX build to work. I primarily play on OSX.

**Comment by lorensghassan** [comment]
Check out the Avalonia branch; UI needs to be moved to cross-platform framework. Discussion is in the #linux section on Discord for SMT. Forked the branch, starting on it this weekend :)

**Comment by lorensghassan** [comment]
Resuming development; got distracted by EVE Online. I will pull latest changes and resume development.

If recollection serves, I need to complete the following items:
1) Check for uses of .NET Framework libraries and move them to .NET Core latest. Integrate back into Avalonia Branch.
2) Complete UI transition to Avalonia.

I will review the source code tonight (I'm PST) and come back with questions for the development team.

**Comment by Slazanger** [comment]
.net framework has already been removed and replaced with core 6.0

**Comment by Slazanger** [comment]
Avalonia has been configured with the docking lib and the individual panels need porting

**Comment by lorensghassan** [comment]
Shows how slow I am :) Downloading GitHub Desktop now and reconfiguring; I was working from the CLI, as I use different tooling my work environment, but this looks to be an easier setup.

**Comment by jmsbooth** [comment]
+++

**Comment by rafaga** [comment]
@Slazanger can you change the reporter of this issue. I'm not interested anymore in following  the porting of this. 

Sorry but I'm doing myself a something similar to SMT in mac (and I'm close to finish that), and now exists RIFT intel fusion tool that does the same in windows/mac/linux. Thanks for your time.

**Comment by Slazanger** [comment]
this is still underway

**Comment by lorensghassan** [comment]
Yes, definitely. In addition to other folks including @Slazanger actively working on this, I am working through matrix transformations to get the pane to move with the window for cross-platform functionality. I will admit its outside my normal specialty, but its an interesting problem space and I will help contribute until it is completed. Hope to have another commit this week, its been a serious pain for me to figure this out.
